src/blueprint/from_config/operators/http.rs

```diff
@@ -30,6 +30,28 @@ impl<'a> MustachePartsValidator<'a> {
   fn new(type_of: &'a config::Type, config: &'a Config, field: &'a FieldDefinition) -> Self {
     Self { type_of, config, field }
   }
+  fn get_nested_type(&self, tail: &[String], mut type_of: &'a config::Type) -> Result<(Type, String), String> {
+    let mut len = tail.len();
+    for item in tail {
+      let val_type = get_value_type(type_of, item).ok_or_else(|| format!("no value '{}' found", item))?;
+
+      if len == 1 {
+        return Ok((val_type, item.to_owned()));
+      } else if val_type.is_nullable() {
```

Comment:
in the last step in `validate` function we check for nullable only for !is_query. Should it be checked the same way here?

Reply:
done